lbz/jwt_utils.py

```diff
@@ -26,16 +26,16 @@ def get_matching_jwk(auth_jwt_token: str) -> dict:
             if key["kid"] == kid_from_jwt_header:
                 return key
 
-        logger.error(f"Required key not found in configuration.")
+        logger.warning("Required key not found in configuration.")
         raise Unauthorized
     except (JWTError, KeyError):
         raise Unauthorized
 
 
 def decode_jwt(auth_jwt_token: str) -> dict:
     if not PUBLIC_KEYS:
-        logger.error("Invalid configuration - no keys in the ALLOWED_PUBLIC_KEYS env variable")
-        raise Unauthorized
+        msg = "Invalid configuration - no keys in the ALLOWED_PUBLIC_KEYS env variable"
+        raise RuntimeError(msg)
```

Comment:
FYI: I have replaced it with `RuntimeError` because it is the server-side problem. We can create even a custom exception for things like that if you want 😉